docs/installation.rst

```diff
@@ -25,9 +25,17 @@ Distribution Package
 --------------------
 
 Some distributions might offer a ready-to-use ``borgbackup``
-package which can be installed with the package manager.  As |project_name| is
-still a young project, such a package might be not available for your system
-yet.
+package which can be installed with the package manager.
+
+.. important:: Those packages may not be up to date with the latest
+               |project_name| releases. Before submitting a bug
+               report, check the package version and compare that to
+               our latest release then review :doc:`changes` to see if
+               the bug has been fixed. Report bugs to the package
+               maintainer instead of directly to |project_name| if the
```

Comment:
No. "instead of" is what i meant.

http://www.thefreedictionary.com/instead+of

I used "rather than" to make it more pleasing.

-- 
Conformity-the natural instinct to passively yield to that vague something
recognized as authority.
                        - Mark Twain